phase/builder_test.go

```diff
@@ -191,8 +191,8 @@ func testBuilder(t *testing.T, when spec.G, it spec.S) {
 					h.AssertContains(t, inputs.TargetEnv, "CNB_TARGET_ARCH=amd64")
 					h.AssertContains(t, inputs.TargetEnv, "CNB_TARGET_ARCH_VARIANT=")
 					h.AssertContains(t, inputs.TargetEnv, "CNB_TARGET_OS=linux")
-					h.AssertContains(t, inputs.TargetEnv, "CNB_TARGET_DISTRO_NAME=")
-					h.AssertContains(t, inputs.TargetEnv, "CNB_TARGET_DISTRO_VERSION=")
+					h.AssertContains(t, inputs.TargetEnv, "CNB_TARGET_DISTRO_NAME=ubuntu")
+					h.AssertContains(t, inputs.TargetEnv, "CNB_TARGET_DISTRO_VERSION=22.04")
```

Comment:
I think these assertions are causing the tests to fail on Windows (and in face would only pass when run on an ubuntu/22.04 system). I think we can just remove them.